src/main/scala/org/broadinstitute/hail/driver/ExportVariantsSolr.scala

```diff
@@ -65,31 +66,17 @@ object ExportVariantsSolr extends Command with Serializable {
     case _ => fatal("")
   }
 
-  def escapeSolrFieldName(name: String): String = {
-    val sb = new StringBuilder
-
-    if (name.head.isDigit)
-      sb += '_'
-
-    name.foreach { c =>
-      if (c.isLetterOrDigit)
-        sb += c
-      else
-        sb += '_'
-    }
-
-    sb.result()
-  }
+  def escapeString(name: String): String =
+    escapeStringSimple(name, '_', !_.isLetter, !_.isLetterOrDigit)
 
   def addFieldReq(preexistingFields: Set[String], name: String, spec: Map[String, AnyRef], t: Type): Option[SchemaRequest.AddField] = {
-    val escapedName = escapeSolrFieldName(name)
-    if (preexistingFields(escapedName))
+    if (preexistingFields(name))
       return None
 
     var m = spec
```

Comment:
Tricky Scala!  Actually no.  `m` is an immutable Map, and below I do `m += ...` which is sugar for `m = m + ...` so it functionally adds the element but then updates `m`.